CMakeLists.txt

```diff
@@ -93,8 +93,10 @@ endif()
 enable_testing()
 
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
-include(googletest)
 include(pybind11)
+include(torch)
```

Comment:
We may also make `k2` avaiable from other DL toolkits (e.g. TensorFlow, MegEngine), I guess it's better if we can make this dependency configurable as users usually would not use  those toolkits together.

Reply:
Yes, I agree. But I think the priority is first to make one framework work, i.e., PyTorch.  Support for multiple
framework in one go may be too complicated.

Reply:
Now the user can choose whether to include PyTorch.